openapi.yml

```diff
@@ -972,7 +972,9 @@ paths:
                     name: limit
                     description: 'Limits the results to a specified number, defaults to 50'
                     schema:
-                        type: integer
+                        type: array
+                        items:
+                            type: string
```

Comment:
what's happening here? looks like you're changing the limit type

Reply:
strange. I'm not sure how it got here